frontend/components/text2text/results/Text2TextList.vue

```diff
@@ -208,9 +187,13 @@ export default {
     },
     selectedSentence() {
       return (
-        this.sentences[this.itemNumber] && this.sentences[this.itemNumber].text
+        this.sentences[this.predictionNumber] &&
+        this.sentences[this.predictionNumber].text
```

Comment:
is this.sentences[this.predictionNumber].text always defined ? if not necesary this.sentences[this.predictionNumber]?.text